Sources/Valet/SecureEnclaveAccessControl.swift

```diff
@@ -22,17 +22,17 @@ import Foundation
 
 
 @objc(VALSecureEnclaveAccessControl)
-public enum SecureEnclaveAccessControl: Int, CustomStringConvertible, Equatable {
+public enum SecureEnclaveAccessControl: Int, CaseIterable, CustomStringConvertible, Equatable {
     /// Access to keychain elements requires user presence verification via Touch ID, Face ID, or device Passcode. Keychain elements are still accessible by Touch ID even if fingers are added or removed. Touch ID does not have to be available or enrolled.
     /// - version: Available on iOS 8 or later, tvOS 8 or later, watchOS 2.0 or later, and macOS 10.11 or later.
     case userPresence = 1
     
     /// Access to keychain elements requires user presence verification via Face ID, or any finger enrolled in Touch ID. Keychain elements remain accessible via Face ID or Touch ID  after faces or fingers are added or removed. Face ID must be enabled with at least one face enrolled, or Touch ID must be available and at least one finger must be enrolled.
-    /// - version: Available on iOS 9 or later, tvOS 9 or later, watchOS 2.0 or later, and macOS 10.12.1 or later.
+    /// - version: Available on iOS 9 or later, tvOS 11.3 or later, watchOS 2.0 or later, macOS 10.12.1 or later, and macOS application extensions in 10.13.4 or later.
```

Comment:
This comment was wrong. Support for this field was added after iOS 9. Here and below.